di_website/datasection/mixins.py

```diff
@@ -29,12 +30,16 @@ class Meta():
     ], blank=True)
     meta_data = StreamField(
         [
-            ('description', RichTextBlock(required=True)),
-            ('provenance', RichTextBlock()),
-            ('variables', RichTextBlock()),
-            ('geography', RichTextBlock()),
-            ('licence', RichTextBlock()),
-            ('citation', RichTextBlock())
+            ('description', TextBlock(required=True)),
+            ('provenance', TextBlock(required=False)),
+            ('variables', TextBlock(required=False)),
+            ('geography', TextBlock(required=False)),
```

Comment:
Was the decision to make them `TextBlock`s for ease of import? Had made them RichText cos the pattern library seemed to have imaged they'd contain more than just text ... maybe some links at least... :)